nav2_behavior_tree/CMakeLists.txt

```diff
@@ -20,6 +20,9 @@ find_package(nav2_util REQUIRED)
 
 nav2_package()
 
+# TODO: Delete after https://github.com/BehaviorTree/BehaviorTree.CPP/issues/811 is released
```

Comment:
File ticket with this commit as a reminder to check in on it, we don't do TODOs in this repo, they're forgotten about easily. Leave the comment though in-line (just not TODO).